pydax/_schema.py

```diff
@@ -108,6 +127,11 @@ class SchemaManager():
 
     def __init__(self, **kwargs: Schema) -> None:
         """Constructor method
+
+        Example:
+
+        >>> dataset_schema = Schema('./tests/schemata/datasets.yaml')
+        >>> schema_manager = SchemaManager(datasets=dataset_schema)
```

Comment:
This should be in class document. docstrings under `__init__` are not used in documentation in pydax (this will become clear after #150 ).